app/services/people/import_from_nomis.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+module People
+  class ImportFromNomis
+    MAPPING = {
+      cro_number: :criminal_records_office,
+      date_of_birth: :date_of_birth,
+      first_name: :first_names,
+      last_name: :last_name,
+      latest_booking_id: :latest_nomis_booking_id,
+      pnc_number: :police_national_computer,
+      prison_number: :prison_number,
+    }.freeze
+
+    def initialize(prison_number)
+      @prison_number = prison_number
+    end
+
+    def call
+      person = Person.find_or_initialize_by(prison_number: @prison_number)
+      person.assign_attributes(person_attributes)
+      person.save!
+    end
+
+  private
+
+    def person_attributes
+      person_attributes = person_from_nomis.filter_map { |attribute, value|
+        [MAPPING[attribute], value] if MAPPING[attribute]
+      }.to_h
+
+      person_attributes.merge(
+        nomis_prison_number: @prison_number,
```

Comment:
I do not think we need `nomis_prison_number`. If we do we should use the MAPPING.

Reply:
Let's do a migration to remove it separately later.